pages/common/tee.md

```diff
@@ -9,3 +9,7 @@
 - Append to the given FILEs, do not overwrite:
 
 `echo "example" | tee -a {{FILE}}`
+
+- Create a folder called "example", print "example" and write "example" to FILE:
+
+`echo "example" | tee >(xargs mkdir) >(cat) >{{FILE}}`
```

Comment:
Perhaps we could change this to `path/to/file`? That's the convention we've used in other pages.